src/data/team-orig.json

```diff
@@ -0,0 +1,319 @@
+
+{
+    "year": "2018-2019",
+    "date": "01/09/2018",
```

Comment:
i don't think you need `date` or `class_name` since those were needed for the carousel

Reply:
wait is there a point for this file? I just saw the other json

Reply:
the other json is the one that the data is being drawn from, team-orig.json is the json that was originally there before i changed things

Reply:
so is it needed in the end?

Reply:
you said to save the original one before i changed it around so besides acting as a backup it's not needed

Reply:
oh i said that? oops yeah we probably don't need it in the repo then